dataland-backend-utils/src/main/kotlin/org/dataland/datalandbackendutils/utils/swaggerdocumentation/BackendOpenApiCustomAnnotations.kt

```diff
@@ -0,0 +1,95 @@
+package org.dataland.datalandbackendutils.utils.swaggerdocumentation
+
+import io.swagger.v3.oas.annotations.Parameter
+
+/** * DataType annotation */
```

Comment:
```suggestion
/** DataType annotation */
```
just looks a bit strange, but no biggie. still, could be bulk changed.

Reply:
done